Crashlytics/Crashlytics/Models/Record/FIRCLSRecordFrame.m

```diff
@@ -16,16 +16,43 @@
 
 #import "FIRCLSRecordFrame.h"
 
+@interface FIRCLSRecordFrame ()
+
+// Internal representation of optional numerical values in Frames
+// These are NSNumber pointers so we can tell when the value doesn't exist
+@property(nonatomic) NSNumber *lineNumber;
+@property(nonatomic) NSNumber *offsetNumber;
```

Comment:
NSNumber is immutable. We should use the same attributes similar to NSString: nonatomic, copy?

Reply:
I think that's right